wmn-data.json

```diff
@@ -3473,6 +3473,16 @@
         "known" : ["test", "janek"],
         "cat" : "images"
        },
+        {
+        "name" : "Moto Trip",
+        "uri_check" : "https://moto-trip.com/profil/{account}",
+        "e_code" : 200,
+        "e_string" : "Profil de:",

```

Comment:
When i look at the response from the server, I do not get `Profil de:` when a good account (pif) is requested. Please check this and modify. Perhaps changing to `h2\">Profil de`